quran/models.py

```diff
@@ -0,0 +1,23 @@
+'''Models describing abstractions used at Quran app'''
+from django.db import models
+import django.contrib.postgres.fields
+from django.contrib.postgres.fields import ArrayField
+
+
+class Surah(models.Model):
+    '''General information about Surahs'''
+    id = models.AutoField(primary_key=True)
```

Comment:
Let's make everything uniform and remove `primary_key=True`.

Reply:
It's good to have primary key in every table, here surah ids are unique and referenced from ayah as ForeignKey, so my guess leave it.

Reply:
Primary key is MUST in every table and if you omit it’s specification, it will be created automatically in your model and called `pk`.